benchmark/Speed.hs

```diff
@@ -131,10 +131,7 @@ samplesBenchmarks e lrData hmmData ldaData = benchmarks
 
 main :: IO ()
 main = do
-  g <- createSystemRandom
-  writeFile "speed-length.csv" ""
-  writeFile "speed-samples.csv" ""
-  writeFile "raw.dat" ""
+  g <- newIOGenM (mkStdGen 1729)
```

Comment:
Can we make a synonym for `newIOGenM (mkStdGen 1729)`. It crops up a lot. Maybe `fixedGen`?

Reply:
Great feedback :-) thanks

Do you have a rationale for the name changes? I am particularly against sis -> sequentially as sis has a particular meaning in the world of SMC.

Reply:
ah ok, I didn't know that. We can leave it